template/package.json

```diff
@@ -1,33 +1,28 @@
 {
   "name": "%TITLE%",
   "version": "1.0.0",
+  "private": true,
```

Comment:
These projects should have always been private, since they are web-apps and not libraries.